packages/scandipwa/src/component/ProductBundleItems/ProductBundleItems.container.js

```diff
@@ -135,6 +149,62 @@ export class ProductBundleItemsContainer extends ProductCustomizableOptionsConta
             );
     }
 
+    resetDefaultSelectOptions() {
+        this.setState({
+            selectedCheckboxValues: [],
+            selectedDropdownOptions: []
+        }, () => this.setDefaultSelectedOptions());
+    }
+
+    setDefaultSelectedOptions() {
+        const { items } = this.props;
+
+        return items.reduce((acc, item) => {
+            const { type } = item;
+
+            switch (type) {
+            case SELECT_TYPE:
+            case RADIO_TYPE: // handle radio as select
+                this.setDefaultDropdownValue(item);
+                break;
+            case CHECKBOX_TYPE:
+            case MULTI_TYPE: // handle multi-select as checkbox
+                this.setDefaultCheckboxValue(item);
+                break;
+            default:
+                return acc;
+            }
+
+            return acc;
+        }, []);
+    }
+
+    setDefaultCheckboxValue(item) {
+        const { option_id, options } = item;
+
+        return options.reduce((acc, { is_default, id, quantity }) => {
+            if (is_default) {
+                const value = id.toString();
+                this.setSelectedCheckboxValues(option_id, { value, quantity });
+            }
+
+            return acc;
+        }, []);
+    }
+
+    setDefaultDropdownValue(item) {
+        const { option_id, options } = item;
+
+        return options.reduce((acc, { is_default, id, quantity }) => {
+            if (is_default) {
+                const value = id.toString();
+                this.setSelectedDropdownValue(option_id, { value, quantity });
+            }
+
+            return acc;
+        }, []);
+    }
```

Comment:
This can be unified into one function, in order to keep things DRY. 
The only difference is the callable function inside of the reduce, and it can be passed as an argument.
Try it, will it be OK-readable? If not - comment on it.
Let's stay concise!

Reply:
Hello! 
I unified functions and seems that logic is pretty easy so I didn't comment it